pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -201,6 +201,27 @@ func (o *GinkgoRunSuiteOptions) Run(suite *TestSuite, junitSuiteName string, mon
 		logrus.Infof("Using built-in tests only due to OPENSHIFT_SKIP_EXTERNAL_TESTS being set")
 	}
 
+	// Temporarily check for the presence of the [Skipped:xyz] annotation in the test names, once this synthetic test
+	// begins to pass we can remove the annotation logic
+	var annotatedSkipped []string
+	for _, t := range externalTestCases {
+		if strings.Contains(t.name, "[Skipped") {
+			annotatedSkipped = append(annotatedSkipped, t.name)
+		}
+	}
+	var skippedAnnotationSyntheticTestResults []*junitapi.JUnitTestCase
+	skippedAnnotationSyntheticTestResult := junitapi.JUnitTestCase{
+		Name: "Skipped annotations present",
```

Comment:
Should we consider [sig-trt] or some annotation to associate this with team or component?